src/jvm/main/org/jetbrains/kotlinx/lincheck/TraceDebugger.kt

```diff
@@ -0,0 +1,13 @@
+/*
+ * Lincheck
+ *
+ * Copyright (C) 2019 - 2025 JetBrains s.r.o.
+ *
+ * This Source Code Form is subject to the terms of the
+ * Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed
+ * with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.jetbrains.kotlinx.lincheck
+
+val isTraceDebuggerEnabled = System.getProperty("lincheck.traceDebuggerMode", "false").toBoolean()
```

Comment:
Let's rename to `inTraceDebuggerMode` or `IN_TRACE_DEBUGGER_MODE`

Reply:
Then change the flag in test configurations, please

Reply:
I mean the `val` name, from `isTraceDebuggerEnabled` to `inTraceDebuggerMode`

Reply:
What about `isInTraceDebuggerMode`?

Reply:
Looks good to me

Reply:
Done